### PR TITLE
Teamsfx toolkit migration nodejs samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/graph-meeting-notification/nodejs/.env [hunk 1]
```diff
@@ -1,7 +1,7 @@
 MicrosoftAppId=<<MicrosoftAppId>>
 MicrosoftAppPassword=<<MicrosoftAppPassword>>
 MicrosoftAppTenantId=<<MicrosoftAppTenantId>>
-notificationUrl=<<BaseUrl>>/api/notifications
+notificationUrl=
```
Comment (ChetanSharma-msft) [inline samples/graph-meeting-notification/nodejs/.env:4]: What should be the value for it?
Kindly mention in the Readme file.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments

File: samples/graph-meeting-notification/nodejs/.env (hunk 1 above)
Comment (Mohammed-MSFT) [reply samples/graph-meeting-notification/nodejs/.env:4]: Updated Readme, please review it once.

**Comment by ChetanSharma-msft** [comment]
Executed build pipeline: https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6171439056

**Comment by ChetanSharma-msft** [comment]
Hello @formulahendry - Could you please review and approve this PR?

**Review by Mohammed-MSFT — APPROVED** [review]
Changes looks fine, Approving it.

**Review by formulahendry — APPROVED** [review]

**Comment by formulahendry** [comment]
Update the root readme as well?
image

**Comment by Harikrishnan-MSFT** [comment]
**Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6244510779**

image

**Pull request merged**